pkgs/top-level/all-packages.nix

```diff
@@ -5528,6 +5530,7 @@ with pkgs;
   lua52Packages = recurseIntoAttrs lua5_2.pkgs;
   lua53Packages = recurseIntoAttrs lua5_3.pkgs;
   lua54Packages = recurseIntoAttrs lua5_4.pkgs;
+  lua55Packages = recurseIntoAttrs lua5_5.pkgs;
```

Comment:
todo: either remove this alias or wait for a luarocks version that supports lua 5.5